frameworks/Swift/swift-nio/benchmark_config.json

```diff
@@ -0,0 +1,24 @@
+{
+  "framework": "swift-nio",
+  "tests": [{
+    "default": {
+      "json_url": "/json",
+      "plaintext_url": "/plaintext",
+      "port": 8080,
+      "approach": "Realistic",
+      "classification": "Platform",
+      "database": "None",
+      "framework": "SwiftNIO",
+      "language": "Swift",
+      "flavor": "None",
+      "orm": "Raw",
+      "platform": "SwiftNIO",
```

Comment:
Similarly, it feels to me like we are using language built-ins rather than a platform for development (see examples like `Gemini` where we use the `Servlet` platform which sits between the language and application code, providing an interface for serving requests that the framework must implement).

I think this should be `"None"`